pkg/store/gcs_test.go

```diff
@@ -274,6 +310,9 @@ func TestGCSBlock_matches(t *testing.T) {
 			matchers: []labels.Matcher{
 				labels.NewEqualMatcher("a", "b"),
 			},
+			expBlockMatchers: []labels.Matcher{
+				labels.NewEqualMatcher("a", "b"),
+			},
 			ok: true,
 		},
```

Comment:
Should we have one more  test case where input matchers do not equal the block matchers?

Reply:
sure, why not

Reply:
done